src/easydiffraction/crystallography/space_group_lookup_table.py

```diff
@@ -0,0 +1,615 @@
+space_group_lookup_table = {
+  (1, None): {'IT_number': 1, 'setting': 1, 'IT_coordinate_system_code': None, 'name_H-M_alt': 'P 1 1 1', 'crystal_system': 'triclinic', 'Wyckoff_positions': {'a': {'multiplicity': 1, 'site_symmetry': '1', 'coords_xyz': ['x,y,z']}}},
```

Comment:
This is a constant, so should be uppercased - maybe `SPACE_GROUPS_LOOKUP_DICT`? This is explicitly a dict, so let's call it that

Reply:
Agreed, makes more sense 👍